packages/@aws-cdk/aws-codedeploy/test/lambda/integ.deployment-group.ts

```diff
@@ -0,0 +1,62 @@
+import cloudwatch = require('@aws-cdk/aws-cloudwatch');
```

Comment:
Can we make this a bit more idiomatic in style so we can use as an example?
Also worth adding to examples I guess...